examples/payment-channel/lib.rs

```diff
@@ -0,0 +1,557 @@
+//! # Payment Channel
```

Comment:
Thanks for adding this. Can you also wrap everything here to 90 characters?

Reply:
It's done.